boards/arm/ip_k66f/ip_k66f_defconfig

```diff
@@ -8,9 +8,18 @@ CONFIG_GPIO=y
 CONFIG_PINMUX=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=180000000
 CONFIG_OSC_LOW_POWER=y
-CONFIG_OSC_XTAL0_FREQ=12000000
-CONFIG_K6X_FLEXBUS_CLOCK_DIVIDER=2
+CONFIG_K6X_BUS_CLOCK_DIVIDER=3
+CONFIG_K6X_FLEXBUS_CLOCK_DIVIDER=3
+CONFIG_K6X_FLASH_CLOCK_DIVIDER=7
 CONFIG_USE_SEGGER_RTT=y
 
+# Add RTT console support
+CONFIG_CONSOLE=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_SEGGER_RTT_BUFFER_SIZE_DOWN=64
+
```

Comment:
Perhaps mention in the board documentation that this board, unlike most if not all others in-tree, default to using Segger RTT for console?

Reply:
I will extend the documentation entry.